content/en/serverless/installation/python.md

```diff
@@ -0,0 +1,187 @@
+
+1. Install the Serverless Plugin in your application package: 
+	`yarn add --dev serverless-plugin-datadog`
+2. In your `serverless.yml`, add the following:
+    ```
+    plugins:
+		    - serverless-plugin-datadog
+    ```
+3. In your `serverless.yml`, also add the following section:
+    ```
+    custom:
+        datadog:
+        flushMetricsToLogs: true
+        # The Datadog Forwarder ARN goes here.
+        Forwarder:
+    ```
+    For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][1].
+4. Redeploy your serverless application.
+
+[1]: https://console.aws.amazon.com/cloudformation/home#/stacks?filteringText=forwarder
+{{% /tab %}}
+{{% tab "SAM" %}}
+
+### Install the Datadog Lambda Library
+
+Use the Datadog SAM macro to ingest traces from your application without any code instrumentation. The macro automatically attaches the Datadog Lambda Library for Node.js and Python to your functions using layers. At deploy time, it generates new handler functions that wrap your existing functions and initializes the Lambda Library.
+
+To install the Datadog Lambda Library with the SAM macro, follow these steps:
+
+1. In your `template.yml`, add the following:
+  ```
+  Transform:
+	  DD::Serverless
+  ```
+2. In your `template.yml`, also add the following section:
+  ```
+  Mappings:
+	  Custom:
+		  Datadog:
+			  flushMetricsToLogs: true
+			  # The Datadog Forwarder ARN goes here.
+			  Forwarder:
+  ```
+  For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][1].
+4. Redeploy your serverless application.
+
+
+[1]: https://console.aws.amazon.com/cloudformation/home#/stacks?filteringText=forwarder
+{{% /tab %}}
+{{% tab "AWS Console" %}}
+
+### Install the Datadog Lambda Library
+
+The Datadog Lambda Library can be imported as a layer. It’s ARN includes a region, language runtime, and version. Construct yours in the following format:
+
+```
+arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-<RUNTIME>:<VERSION>
+```
+
+For example:
+
+```
+arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Python37:11
+```
+
+The Python Datadog Lambda Library supports Python27, Python36, Python37, and Python38. For more information, see the [latest release][1].
+
+To install the Datadog Lambda Library:
+
+1. Navigate to the Lambda function to which you want to add the layer in your AWS console.
+2. Click on **Layers** on the main page of your function.
+3. Scroll down, and click on **Add a Layer**.
+4. Select the option to *Provide a layer version ARN*.
+5. Enter the Datadog Lambda Library ARN from the table above.
+6. Navigate to the Environment Variables section of your function and add a new `“DD_FLUSH_TO_LOG”` variable set to `“true”`.
+
+These steps need to be repeated for every function you wish to trace.
+
+### Instrument your Code
```

Comment:
@apigirl Do you prefer "Instrument your Code" or "Instrument your Application"?

Reply:
let's leave it code for now because it doesn't actually have to be an entire application